docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.4'
```

Comment:
können wir das stattdessen direkt in `hitobito/development:/docker-compose.yml` pflegen?